test/test.js

```diff
@@ -1,234 +0,0 @@
-const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
```

Comment:
Let the HTML version of testing exist for sometime

Reply:
If this is about old integration/browser test, don't remove it. Move it to `integrationtests` folder or something.

Reply:
Created separate branch for the change AR-1880-integration-tests